drivers/overlay/overlay.go

```diff
@@ -473,22 +592,42 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return mergedDir, nil
 	}
 	defer func() {
-		if err != nil {
-			if c := d.ctr.Decrement(mergedDir); c <= 0 {
-				syscall.Unmount(mergedDir, 0)
+		if retErr != nil {
+			if err != nil {
```

Comment:
We're switching from checking `err` to checking `retErr` here, so this line and its matching closing brace shouldn't be here.

Reply:
Yup I screwed this patch up.  Will push the correct change from upstream.